src/alg_utils.jl

```diff
@@ -9,6 +9,7 @@ isfsal(alg::FunctionMap) = false
 isfsal(alg::Rodas4) = false
 isfsal(alg::Rodas42) = false
 isfsal(alg::Rodas4P) = false
+isfsal(alg::RosenbrockW6S4OS)=false
```

Comment:
I don't see a good reason why this shouldn't use FSAL for a 3rd order interpolation?

Reply:
I thought it need `c6=1` in `du = f(u, p, t+c6*dt)` to have FSAL property?